code/modules/awaymissions/mission_code/dsbssigma.dm

```diff
@@ -0,0 +1,666 @@
+		if(did_upgrade)
+			to_chat(user, "<span class='notice'>An access type was added to your ID card. You think you will be able to lift the lockdown with this.</span>")
+		else
+			to_chat(user, "<span class='notice'>Your ID card already has all the access this machine can give.</span>")
+		return
+	return ..()
+
+/obj/machinery/vending/medical/syndicate_access/dsbssigma
+	req_access = list(ACCESS_AWAY01)
+	products = list(/obj/item/reagent_containers/syringe = 4, /obj/item/reagent_containers/food/pill/patch/styptic = 2, /obj/item/reagent_containers/food/pill/patch/silver_sulf = 2, /obj/item/reagent_containers/applicator/brute = 1, /obj/item/reagent_containers/applicator/burn = 1,
+					/obj/item/reagent_containers/glass/bottle/charcoal = 1, /obj/item/reagent_containers/glass/bottle/salicylic = 1, /obj/item/reagent_containers/glass/bottle/saline = 1, /obj/item/reagent_containers/glass/bottle/ether = 1,
+					/obj/item/reagent_containers/glass/bottle/atropine = 1, /obj/item/reagent_containers/glass/bottle/oculine = 1, /obj/item/reagent_containers/glass/bottle/toxin = 1, /obj/item/reagent_containers/syringe/antiviral = 2,
+					/obj/item/reagent_containers/syringe/insulin = 1, /obj/item/reagent_containers/hypospray/autoinjector = 2, /obj/item/reagent_containers/food/pill/salbutamol = 2, /obj/item/reagent_containers/food/pill/mannitol = 2, /obj/item/reagent_containers/food/pill/mutadone = 1,
+					/obj/item/stack/medical/bruise_pack/advanced = 1, /obj/item/stack/medical/ointment/advanced = 1, /obj/item/stack/medical/bruise_pack = 2, /obj/item/stack/medical/splint = 2, /obj/item/reagent_containers/glass/beaker = 2, /obj/item/reagent_containers/dropper = 1,
+					/obj/item/healthanalyzer = 1, /obj/item/healthupgrade = 1, /obj/item/reagent_containers/iv_bag/slime = 1)
+	contraband = list(/obj/item/reagent_containers/glass/bottle/sulfonal = 1, /obj/item/reagent_containers/glass/bottle/pancuronium = 1)
+
+/obj/machinery/autolathe/hacked
+	hacked = TRUE
+
+//////// MECHA ////////
+
+/obj/mecha/combat/gygax/dark/rusted
+	desc = "This once lightweight exosuit, painted in a dark scheme, has not been spared by the elements. Exterior armor is heavily rusted and most equipment sockets have decayed beyond repair."
+	name = "Rusted Dark Gygax"
+	max_integrity = 200
+	deflect_chance = 10
+	armor = list(melee = 20, bullet = 20, laser = 25, energy = 20, bomb = 10, bio = 0, rad = 10, fire = 50, acid = 50)
+	max_temperature = 25000
+	leg_overload_coeff = 50
+	operation_req_access = list(ACCESS_AWAY01)
+	internal_damage_threshold = 25
+	max_equip = 0
+	destruction_sleep_duration = 2
+
+/obj/mecha/combat/gygax/dark/rusted/add_cell()
+	cell = new /obj/item/stock_parts/cell/high(src)
+
+/obj/mecha/combat/gygax/dark/rusted/onTransitZ()
+	do_sparks(3, 1, src)
+	take_damage(300, BRUTE)
+
+////// STRUCTURES //////
+
+/obj/structure/displaycase/telecrystal/dsbssigma
+	req_access = list(ACCESS_AWAY01)
+
+/////// SPAWNERS ///////
+
+/obj/effect/landmark/away
```

Comment:
I suggest you set the name of this in the code so that for whatever reason in the future, you, or someone else doesn't have to set the name for these manually while mapping.
```suggestion
/obj/effect/landmark/away
	name = "dsbssigma_queen_spawn"
```
You can even include the name in the type path if you want:
```suggestion
/obj/effect/landmark/away/queen_spawn
	name = "dsbssigma_queen_spawn"
```